src/SFA.DAS.ApplyService.Web/Services/TrusteeExemptionService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using SFA.DAS.ApplyService.Configuration;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApplyService.Web.Services;
+
+public class TrusteeExemptionService : ITrusteeExemptionService
+{
+    private readonly IConfigurationService _configurationService;
+
+    public TrusteeExemptionService(IConfigurationService configurationService)
+    {
+        _configurationService = configurationService;
+    }
+
+    public async Task<bool> IsProviderExempted(string ukprn)
+    {
+        var config = await _configurationService.GetConfig();
```

Comment:
Can we not use IOptions to load this value at the start up so that config is loaded only once per application lifecycle?

Reply:
_configureService is already cached. Isn't that the same?